src/Form.js

```diff
@@ -100,6 +100,25 @@ export default class Form {
   }
 
   /**
+   * Checks if all fields have a value.
+   * @type {boolean}
+   */
+  @computed get isComplete (): boolean {
+    const value = _.find(this.fields, (field: Field) => {
```

Comment:
This will iterate over all fields. You can use [_.some](https://lodash.com/docs/4.17.4#some) to run only until the first field not completed